spacy/tests/test_cli.py

```diff
@@ -855,3 +857,35 @@ def test_span_length_freq_dist_output_must_be_correct():
     span_freqs = _get_spans_length_freq_dist(sample_span_lengths, threshold)
     assert sum(span_freqs.values()) >= threshold
     assert list(span_freqs.keys()) == [3, 1, 4, 5, 2]
+
+
+def test_project_check_requirements():
+    reqs = """
+    spacy
+
+    # comment
+
+    thinc"""
+    output = _check_requirements([req.strip() for req in reqs.split("\n")])
+    assert output == (False, False)
+
+    reqs = """# comment
+    --some-flag
+    spacy"""
+    output = _check_requirements([req.strip() for req in reqs.split("\n")])
+    assert output == (False, False)
+
+    reqs = """# comment
+    --some-flag
+    spacy; python_version >= '3.6'"""
+    output = _check_requirements([req.strip() for req in reqs.split("\n")])
+    assert output == (False, False)
+
+    # excessive guard against unlikely package name
+    try:
+        pkg_resources.require("spacyunknowndoesnotexist12345")
+    except pkg_resources.DistributionNotFound:
+        reqs = """# comment
+        spacyunknowndoesnotexist12345"""
+        output = _check_requirements([req.strip() for req in reqs.split("\n")])
+        assert output == (True, False)
```

Comment:
The point of the `try`/`except` is to verify first that the unlikely package is not installed, right? Alternative suggestion, feel free to ignore:
```suggestion
    pkg_name = "spacyunknowndoesnotexist12345"
    with pytest.raises(pkg_resources.DistributionNotFound):
        pkg_resources.require(pkg_name)
    reqs = f"# comment\n{pkg_name}"
    output = _check_requirements([req.strip() for req in reqs.split("\n")])
    assert output == (True, False)
```

Reply:
I might be confused about what you're trying to propose, but the point of this was to completely skip the test in the (rare, unlikely) case that someone intentionally created an venv with this package in order to break the test suite.

Reply:
Ah, I thought we wanted to guarantee that this package is not installed. Skipping makes more sense.